lib/hooks/prerun/track.ts

```diff
@@ -16,8 +16,12 @@
  */
 import { Hook } from '@oclif/config';
 
+let trackResolve: (result: Promise<any>) => void;
```

Comment:
If you want to be fully accurate you can type this as
```suggestion
let trackResolve: ConstructorParameters<PromiseConstructor>[0];
```

Reply:
In this case it may be better not to be fully accurate, :-) because the suggested alternative type is somewhat obscure. It is the same type as that of the resolve argument to `Promise((resolve, reject) => ...)`, which my VS Code IDE tells to be `(value?: unknown) => void`. I replaced `unknown` with a less unknown type.